src/test/java/seedu/address/logic/commands/AddAppointmentCommandTest.java

```diff
@@ -85,15 +102,6 @@ public void execute_invalidBuyerRole_failure() {
         assertCommandFailure(addAppointmentCommand, model, AddAppointmentCommand.MESSAGE_INVALID_BUYER_ROLE);
     }
 
```

Comment:
Here is another test case to check the validity of sellerRole

```suggestion
@Test
public void execute_invalidSellerRoleWithoutBuyer_failure() {
    // Select a person who is buyer (not a seller)
    Person buyer = PersonBuilder(model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased())).build();
    AddAppointmentCommand addAppointmentCommand = new AddAppointmentCommand(
            new AppointmentDatetime(VALID_APPOINTMENT_DATETIME_JAN_1), INDEX_FIRST_PERSON);
            
    assertCommandFailure(addAppointmentCommand, model, AddAppointmentCommand.MESSAGE_INVALID_BUYER_ROLE);
```

Reply:
this would actually result in invalid seller role since `INDEX_FIRST_PERSON` is a buyer! No harm having more tests, but I felt that having the wrong seller role with no buyer would not be significantly different from having the wrong seller role and having a buyer.